src/preview/src/components/AdPreview.js

```diff
@@ -114,12 +115,23 @@ export const AdPreview = (props) => {
           : <></>
         }
         {
-          ad.output?.zip?.size
+          ad.output?.zip?.size || ad.output?.unzip?.size
           ? <>
-              <Box marginBottom="20px">
-                <Tooltip title="Bundle size">
-                  <Chip icon={Math.round(ad.output.zip.size / 1024) <= maxFileSize ? <DoneIcon /> : <ClearIcon />} label={`${Math.round(ad.output.zip.size / 1024)} KB`} color={Math.round(ad.output.zip.size / 1024) <= maxFileSize ? "success" : "error"} />
-                </Tooltip>
+              <Box marginBottom="20px" display="flex" flexWrap="wrap" gap="10px">
+                {
+                  ad.output?.zip?.size
+                  ? <Tooltip title="Bundle size">
+                      <Chip icon={Math.round(ad.output.zip.size / 1024) <= maxFileSize ? <DoneIcon /> : <ClearIcon />} label={`${Math.round(ad.output.zip.size / 1024)} KB`} color={Math.round(ad.output.zip.size / 1024) <= maxFileSize ? "success" : "error"} />
+                    </Tooltip>
+                  : <></>
+                }
+                {
+                  ad.output?.unzip?.size
+                  ? <Tooltip title="UnZIP size">
```

Comment:
![image](https://github.com/mediamonks/display-dev-server/assets/45459258/1418db81-dee1-4ed0-b81f-022c84110f61)
To make both tooltips more clear I think it should say "Compressed size" and "Uncompressed size" instead of Bundle size and UnZIP size

Reply:
hey @mm-paulie 

- added a warning message when a bundle wasn't optimized enough, also removed some redundant ones

- and re quality chip: It can be either a weapon against us or our weapon. We can argue with a client about size limits with more confidence and with real proof.
It was previously in the console and not really useful, especially when creating tons of banners. So having it visible for each banner is def better for perception. I can put it into the same route as gsdevtools but I believe it's good to have it at the place where it is now